recipes/heliopy/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "HelioPy" %}
+{% set version = "0.2" %}
+{% set sha256 = "cb94d5ee10c43054a4111517ee0aa513e67030785ab775c2bcd914f9fab7bac9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - pandas
+  test:
+    imports:
+      - heliopy
+      - heliopy.data
+      - heliopy.constants
+      - heliopy.plot
+      - heliopy.util
+      - pycdf.pycdf
+
+about:
+  home: http://heliopy.org/
+  license: GPL-3.0
+  license_family: GPL
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
It looks like in the `0.2` release, the `license_file` is in the source distribution, so if that's the case, you can remove the license that's alongside this recipe and just use `license_file: LICENSE`.